pkg/crawler/crawler.go

```diff
@@ -176,9 +179,19 @@ func (c *Crawler) crawlSHA1(baseURL string, meta *Metadata) error {
 			return err
 		}
 		if len(sha1) != 0 {
+			// fetch licenses
+			pomFileName := fmt.Sprintf("/%s-%s.pom", meta.ArtifactID, version)
+			pomURL := baseURL + version + pomFileName
+			pomLicense, err := c.fetchPOMLicense(pomURL)
+			if err != nil {
+				// TODO: Check if we can change this log to a warning or ignore it
+				log.Println(err)
```

Comment:
I think we can return error here. If pom.xml file doesn't exist - we handle this inside `fetchPOMLicense`.

Reply:
Didn't wanna return the error since version struct is being updated post this step

```
			v := Version{
				Version: version,
				SHA1:    sha1,
				License: pomLicense,
			}
```

Reply:
I'm worried that we might skip this error, because in order to find this error, you need to ope GitHub action logs (in this case, action will succeed).

I think it's better to get error and skip it temporarily if we can't fix it quickly than miss that bug.

Reply:
I had a look at the errors. I have already handled the encoding related errors. In all others scenarios, the pom.xml is invalid.

Log dump for you reference:

[log.txt](https://github.com/aquasecurity/trivy-java-db/files/12288343/log.txt)


Don't think there is much we can do about those. If we decide to return , then we won't be saving the artifact i.e GAV information. Won't that be a problem?

Reply:
I see it now. Thanks for log!

Let's use your solution.